var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -294,3 +294,41 @@
     #
     - { key: 'AutoUpdate.Enabled', value: 'n' }
   when: platform == "azure"
+
+#
+# Customize the GCP linux environment.
+#
+# Update the override file for the GCP instance. This file gets
+# applied dynamically by running google_instance_setup script.
+#
+- blockinfile:
+    path: /etc/default/instance_configs.cfg.template
+    create: yes
+    block: |
+      #
+      # Disable the accounts daemon to prevent adding/removing
+      # users on the engine.
+      #
+      [Daemons]
+      accounts_daemon = false
+
+      #
+      # Disable user supplied startup/shutdown scripts from running on
+      # the engine.
+      #
+      [MetadataScripts]
+      shutdown = false
+      startup = false
+  when:
+    - platform == "gcp"
+    - not ansible_is_chroot
+  notify: "gcp config changed"
+
+#
+# We don't want to run the account daemon when we first boot up so
+# disable here.
+#
+- command: systemctl disable google-accounts-daemon.service
+  when:
+    - platform == "gcp"
+    - ansible_is_chroot
```

Comment:
is this supposed to be `not ansible_is_chroot`?

Reply:
The intent was to only run this at when appliance-build ran and I was trying to be more explicit about when it ran. As Prakash points out, we can just remove this and let it run all the time.